README.md

```diff
@@ -33,13 +36,29 @@ Column {
 }
 ```
 
+Or Add the `HarmonyColorPicker` to your Compose hierarchy for a HSV circle color wheel implementation:
+
+```kotlin
+ HarmonyColorPicker(
+    harmonyMode = harmonyMode.value,
+    modifier = Modifier.size(400.dp),
+    onColorChanged = { hsvColor ->
+        currentColor.value = hsvColor.toColor()
+        extraColors.value = hsvColor.getColors(colorHarmonyMode = harmonyMode.value) 
+})
+```
+
+The `HarmonyColorPicker` allows for you to set a certain `ColorHarmonyMode` to enable the different options to be 
```

Comment:
[nit] I'd rephrase this. It's not just 'options displayed', it's a whole new feature that deserves a proper description.